bin/tunnel_test.py

```diff
@@ -0,0 +1,53 @@
+import logging
```

Comment:
I thought with conda/pyproject.toml we are now supposed to do "bin" scripts differently, no? Shouldn't this be a method inside the module and defined in the pyproject.toml file instead?